frontend/mulighetsrommet-veileder-flate/src/RoutesConfig.tsx

```diff
@@ -1,14 +1,20 @@
 import { Navigate, Route, Routes } from 'react-router-dom';
 import { ViewTiltaksgjennomforingDetaljerContainer } from './views/tiltaksgjennomforing-detaljer/ViewTiltaksgjennomforingDetaljerContainer';
 import ViewTiltaksgjennomforingOversikt from './views/tiltaksgjennomforing-oversikt/ViewTiltaksgjennomforingOversikt';
+import { Toggles } from 'mulighetsrommet-api-client';
+import { useFeatureToggle } from './core/api/feature-toggles';
+import { Landingsside } from './views/landingsside/Landingsside';
 
 const RoutesConfig = () => {
+  const enableLandingssideFeature = useFeatureToggle(Toggles.MULIGHETSROMMET_VEILEDERFLATE_LANDINGSSIDE);
+  const enableLandingsside = enableLandingssideFeature.isSuccess && enableLandingssideFeature.data;
   return (
     <Routes>
-      <Route index element={<ViewTiltaksgjennomforingOversikt />} />
       <Route path="tiltak/:tiltaksnummer" element={<ViewTiltaksgjennomforingDetaljerContainer />} />
       <Route path=":tiltaksnummer" element={<Navigate to="/" />} />
-      <Route path="*" element={<Navigate to="/" />} />
+      <Route path="oversikt" element={<ViewTiltaksgjennomforingOversikt />} />
+      {enableLandingsside ? <Route index element={<Landingsside />} /> : null}
+      <Route path="*" element={<ViewTiltaksgjennomforingOversikt />} />
```

Comment:
Nå er det to routes som rendrer samme komponent `ViewTiltaksgjennomforingOversikt`. Er dette bedre ennn å kjøre en `<Navigate to="/oversikt" />` som nå er valgt som ny url path?

Reply:
Godt spørsmål. Jeg vet ikke hva forskjellen er om vi har en catch-all path `* ` som rendrer oversikten, eller om catch-all navigerer til /oversikt. Jeg kan endre til å navigere til `/oversikt`.

Reply:
Eller, ser at vi ikke kan bytte til `<Navigate` fordi det er en egentlig en funksjon som kalles uansett...